server/src/main/java/io/seata/server/ParameterParser.java

```diff
@@ -0,0 +1,116 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.server;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+
+/**
+ * The type parameter parser
+ *
+ * @author xingfudeshi@gmail.com
+ * @date 2019/05/30
+ */
+public class ParameterParser {
+    private static final String PROGRAM_NAME = "sh seata-server.sh";
+    private static final int SERVER_DEFAULT_PORT = 8091;
+    private static final String SERVER_DEFAULT_BIND_IP = "0.0.0.0";
+    private static final String SERVER_DEFAULT_STORE_MODE = "file";
+
+
+    @Parameter(names = "--help", help = true)
+    private boolean help;
+    @Parameter(names = {"--host", "-h"}, description = "The host to bind.", order = 1)
+    private String host = SERVER_DEFAULT_BIND_IP;
+    @Parameter(names = {"--port", "-p"}, description = "The port to listen.", order = 2)
```

Comment:
The order of the parameters has changed, and the user may not start up using the old command.

Reply:
> The order of the parameters has changed, and the user may not start up using the old command.

Do you mean the order attribute in the @Parameter?It's only the order for displaying of each option.not the parameter passing order.
when type seata-server --help,will display result as following:
![image](https://user-images.githubusercontent.com/24457941/58619256-23c29b00-82f7-11e9-9af1-9618374a7470.png)

Reply:
> > The order of the parameters has changed, and the user may not start up using the old command.
> 
> Do you mean the order attribute in the @parameter?It's only the order for displaying of each option.not the parameter passing order.
> when type seata-server --help,will display result as following:
> ![image](https://user-images.githubusercontent.com/24457941/58619256-23c29b00-82f7-11e9-9af1-9618374a7470.png)

ok, I misunderstand.